src/components/graph/graph.helper.js

```diff
@@ -32,6 +32,9 @@ import ERRORS from "../../err";
 
 import { isDeepEqual, isEmptyObject, merge, pick, antiPick, throwErr, logWarning } from "../../utils";
 import { computeNodeDegree } from "./collapse.helper";
+import { select as d3Select } from "d3-selection";
+import * as d3 from "d3";
```

Comment:
```suggestion
```

Reply:
I need to import 'd3' library because of  d3.zoomIdentity function in row 344- graph.helper.